app/helpers/documents_helper.rb

```diff
@@ -53,11 +53,19 @@ def delete_formatter(document)
     if document.completed? || document.failed?
     [
       "<a class='remove remove-document' href='javascript:void(0)' title='Remove' data-document_id='#{document.id}' data-documentable_type='#{document.documentable_type}'>",
-      "<i class='glyphicon glyphicon-remove'></i>",
+      "<i class='far fa-trash-alt'></i>",
       "</a>"
     ].join ""
     else
       "<i class='glyphicon glyphicon-remove' style='cursor:default'></i>"
     end
   end
+
+  def read_formatter(date)
```

Comment:
Can you YMLize this text? Also I noticed that clicking the download button doesn't update the table to go from "Unread" to "Read". I had to manually refresh

Reply:
Text is ymlized and table refresh is fixed.